src/Generics/SOP/Constraint.hs

```diff
@@ -42,6 +42,24 @@ import Generics.SOP.Sing
 --
 -- means that 'f' can assume that all elements of the n-ary
 -- product satisfy 'Eq'.
+
```

Comment:
The empty line breaks haddock

Reply:
Many apologies, I don't have haddocks generation as part of my workflow. I'll set it up now and update the PR when I've checked my new changes.